scripts/ci/gke.sh

```diff
@@ -101,6 +104,15 @@ create_cluster() {
         labels="${labels/%-/x}"
         labels="${labels},stackrox-ci-build-id=${BUILD_ID:0:63}"
         labels="${labels/%-/x}"
+    elif is_GITHUB_ACTIONS; then
+        require_environment "GITHUB_JOB"
+        require_environment "GITHUB_RUN_ID"
+        tags="${tags},stackrox-ci-${GITHUB_JOB:0:50}"
+        tags="${tags/%-/x}"
+        labels="${labels},stackrox-ci-job=${GITHUB_JOB:0:63}"
+        labels="${labels/%-/x}"
+        labels="${labels},stackrox-ci-build-id=${GITHUB_RUN_ID:0:63}"
+        labels="${labels/%-/x}"
```

Comment:
How about introducing intermediate variables for job name and run id, common for all branches here, and then have a single common piece of code for squeezing them into tags and labels, after the `fi`? Similar to how we define `$build_num` in `assign_env_variables`.

This way we would not repeat the logic that does truncation and replacement of trailing dash. And then we could maybe even add a comment on what is happening, because `${foo/%-/x}` is not exactly obvious to a non-bash-expert and the reason for truncation similarly (is it because we're making assumptions about the structure of the value, or just GKE has a length limitation).